### ISSUE TITLE
Detailed documentation is needed for this great android library !!

### ISSUE THREAD

**Issue by ranaaditya** [linkedin/LiTr#141]
Hi LinkedIn team, you guys are doing awesome by open sourcing some of the great projects, we are thankful to you for all your efforts. Just a small feedback on this android lib - **It would be better if you can provide a detailed documentation of this project, it will help us to get to know which class / object is responsible for what.**

Thanks a lot !

**Comment by RishabJaiswal** [comment]
Agree, this library is great & it needs a better documentation support
